inference-engine/src/vpu/graph_transformer/src/stages/strided_slice.cpp

```diff
@@ -11,22 +11,30 @@ namespace vpu {
 
 namespace {
 
-int maskStrToInt(std::string mask) {
-    int idx = 0, result = 0;
+int maskStrToInt(std::string mask, int elemCount) {
```

Comment:
There are no reverse on device side. Device just reads the appropriate number of mask elements (according to `begin` and `end`) starting from the last bit of mask. So here I tried to achieve this order, but anyway it can be done without any reverse. Fixed.